metrics/jointcal.yaml

```diff
@@ -24,19 +24,19 @@ selected_astrometry_fittedStars:
   unit: ''
   tags: astrometry
 
-selected_astrometry_ccdImages:
+selected_astrometry_ccdImageList:
```

Comment:
This one was my bad: I should have changed these metrics in jointcal to be `*_ccdImages`. Do you want to fix those?